capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -118,8 +118,22 @@ class ClubPage extends Component {
   render() {
     const isOwner = this.state.owner && this.state.club.ownerID === window.localStorage.getItem('userID');
     const bookTile = this.state.book.authors && <BookSearchTile book={this.state.book} bookLists={this.props.bookLists} updateBookLists={this.props.updateBookLists} />;
-    const owner = this.state.owner && <UserCard removeMember={this.removeMember} club={this.state.club} user={this.state.owner} />;
-    const members = this.state.members.length && this.state.members.map(m => <UserCard key={m.id} user={m} club={this.state.club} removeMember={this.removeMember} />);
+    const owner = this.state.owner &&
```

Comment:
nice styling fix, but might conflict when other PRs are merged (since one of mine also alters this). Might be worth reverting and letting other PRs deal with it

Reply:
Ok! I'll move it back. It looks like yours has been merged in already so I can deal with the conflicts